event/tests/test_forms.py

```diff
@@ -33,4 +42,24 @@ def test_create_event_form_invalid(self):
         invalid_data = self.form_data.copy()
         invalid_data.pop("name")
         form = CreateEventForm(data=invalid_data)
+        self.assertFalse(form.is_valid())
+
+    def test_create_group_event_form_valid_data(self):
```

Comment:
Why don't you make a separate class CreateGroupEventForm?

Reply:
It's going to use almost same setup except Group objects is needed.
and it is same app. so I put them in together. so that I don't have to duplicate code.
But I think there is ways that we could make that separate class and reduce duplication. 
I will do that.